CMakeLists.txt

```diff
@@ -182,6 +183,10 @@ include(util)
 # now that all modules and settings are loaded,
 # apply those to the project.
 
+##################################################
+# set project version
+project(openage VERSION "${PROJECT_VERSION}")
```

Comment:
What do you mean? As I wrote it, the variable project_version right now gets the `0.4.0` and converts it automatically into major, minor and patch - by making use of that projects command VERSION parameter. Like this is the status quo with this PR. I didn't make use of tweak till now, because i didn't know what to use there.

Reply:
We could even think about moving this version detection stuff a bit higher in the cmakelists, as stated here:

```
Note Call the project() command near the top of the top-level CMakeLists.txt, but after calling cmake_minimum_required(). It is important to establish version and policy settings before invoking other commands whose behavior they may affect. See also policy CMP0000.
```